test/unit/meta/traits/same_lanes_or_scalar.cpp

```diff
@@ -0,0 +1,43 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+**/
+//==================================================================================================
+#include "test.hpp"
```

Comment:
If the implementations are in the same, so should be tests I think

Reply:
True